packages/main/src/ComboBox.js

```diff
@@ -934,6 +934,7 @@ class ComboBox extends UI5Element {
 		const sameSelectionPerformed = this.value.toLowerCase() === this.filterValue.toLowerCase();
 
 		if (sameItemSelected && sameSelectionPerformed) {
+			this._fireChangeEvent(); // Click on an already typed, but not memoized value shouold also trigger the change event
```

Comment:
"should" is spelled incorrectly in the comment

Reply:
"memorized" too

Reply:
I think there is a term "[memoized](https://en.wikipedia.org/wiki/Memoization)"